src/test/k6/tests/find-tests.js

```diff
@@ -0,0 +1,38 @@
+import http from 'k6/http';
+import { browser } from 'k6/browser';
+import { sleep } from 'k6';
+
+export async function searchResultsTest() {
+    const page = await browser.newPage();
+
+    try {
+        await page.goto('https://test.find-support-for-your-family.education.gov.uk/');
```

Comment:
We can configure these tests to run locally but they are hard coded to run against the live test site here, does that option mean something different?